torch/csrc/Module.cpp

```diff
@@ -369,6 +368,10 @@ IMPLEMENT_STATELESS_ADDXX(baddbmm)
 IMPLEMENT_STATELESS_ADDXX(addcmul)
 IMPLEMENT_STATELESS_ADDXX(addcdiv)
 
+// For torch.polygamma(n, x), the `self` argument comes second, the
+// first two arguments needs to be swapped before dispatch.
+IMPLEMENT_STATELESS_ADDXX(polygamma)
```

Comment:
This macro (`IMPLEMENT_STATELESS_ADDXX`) mentions that it's used for deprecated `add**` functions, which is no longer true. Can you mention that, and possibly rename it (e.g. to `IMPLEMENT_STATELESS_SWAP`) to make it clear that it shouldn't go away?

Reply:
I have add a macro `IMPLEMENT_STATELESS_SWAP` and clarify the it's usage in comment.

Now `IMPLEMENT_STATELESS_ADDXX` is an alias of `IMPLEMENT_STATELESS_SWAP` for deprecated `add**` methods.